DS and Algorithm/Selenium/SeleniumExample.java

```diff
@@ -0,0 +1,23 @@
+// src/main/java/SeleniumExample.java
+public class SeleniumExample {
+    public static void main(String[] args) {
+        // Initialize WebDriver
+        WebDriver driver = WebDriverSetup.initializeChromeDriver();
```

Comment:
**Scalability Issue**: The WebDriver instance is initialized directly in the main method, which can lead to scalability issues when multiple instances are needed simultaneously. <br> **Fix**: Refactor the WebDriver initialization into a singleton pattern or use a WebDriver pool to manage multiple instances efficiently. <br> **Code Suggestion**: 
 ```
 - WebDriver driver = WebDriverSetup.initializeChromeDriver();
 + WebDriver driver = WebDriverPool.getDriverInstance();
 ```